src/Application.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/*
+ * This file is part of the Manalize project.
+ *
+ * (c) Manala <contact@manala.io>
+ *
+ * For the full copyright and license information, please refer to the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Manala\Manalize;
+
+use Symfony\Component\Console\Application as BaseApplication;
+
+/**
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class Application extends BaseApplication
+{
+    const VERSION = '0.2.0-DEV';
+    const COMPOSER_NAME = 'manala/manalize';
```

Comment:
This looks like it is not used for now, but can be renamed and used accordingly to https://github.com/manala/manalize/pull/53#discussion_r82506950